app/scripts/modules/amazon/src/loadBalancer/AmazonLoadBalancerClusterContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import autoBindMethods from 'class-autobind-decorator';
+
+import { ILoadBalancerClusterContainerProps, LoadBalancerClusterContainer } from '@spinnaker/core'
+
+import { IAmazonApplicationLoadBalancer } from '../domain/IAmazonLoadBalancer';
+import { TargetGroup } from './TargetGroup';
+
+interface IState {
+}
+
+@autoBindMethods
+export class AmazonLoadBalancerClusterContainer extends React.Component<ILoadBalancerClusterContainerProps, IState> {
+  constructor(props: ILoadBalancerClusterContainerProps) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  public render(): React.ReactElement<AmazonLoadBalancerClusterContainer> {
```

Comment:
I don't remember this... However, if we ever turn on the tslint rule to require return types, it wouldn't pass lint if we omitted return types from these...